README.md

```diff
@@ -1,56 +1,102 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
 
-> Description the project.
+# budget masive
 
+A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
 ## Built With
+- Ruby on Rails
 
-- Major languages
-- Frameworks
-- Technologies used
+## Heroku
+coming soon
```

Comment:
- Let's add a link to the deployed project as requested in the project requirements.